holoviews/core/accessors.py

```diff
@@ -566,7 +566,7 @@ def __call__(self, *args, **kwargs):
                        "group (i.e. separate plot, style and norm groups) is deprecated. "
                        "Use the .options method converting to the simplified format "
```

Comment:
I'm never sure, what do we recommend usually between calling `.opts` and `.options`?

Reply:
I think the reason `.options` was recommended was because it isn't in-place and is therefore the right analogue to the old syntax.

However, we generally do recommend `.opts` so I think I agree - we should suggest `.opts` here instead of `.options`. Not entirely clear cut though!